admin-dev/themes/new-theme/.webpack/prod.js

```diff
@@ -30,6 +30,13 @@ function prodConfig(analyze) {
     })
   );
 
+  // Required for Vue production environment
```

Comment:
Why it's required? We already have vue app not requiring it :thinking:

Reply:
Better editing the `npm run build` command in the package.json. It's not a good idea to force an env variable in the code
```
"build": "NODE_ENV=production webpack --bail --progress --mode production",
```

Reply:
It's weird if you have the problem and you're the only one with it :sweat_smile:  I tried and can't reproduce :/

Reply:
Mmm, indeed it's weird Here's how I tested, I removed this piec of code, I ran `npm run build` and then went to a page with vue component (currency, translation, ...) And then I saw the warning `You are running Vue in development mode...` in the console
Did you perform the same test? What could be wrong with in my case? I have `npm 6.4.1` and `node v8.14.0`

Reply:
You can use another way to achieve that, as Pierre said, --mode option can be retrieved in webpack.config.js if you want : https://github.com/webpack/webpack/issues/7074#issuecomment-399853701

Reply:
`--mode` is not available for webpack 2 :sob:

Reply:
`--mode` doesn't work with this version of webpack